internal/routes/routes.go

```diff
@@ -0,0 +1,39 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/pyramid.io/planit-backend/internal/handlers/dummy_handler"
+	json_handler "github.com/pyramid.io/planit-backend/internal/handlers/json_handler"
+	"github.com/pyramid.io/planit-backend/pkg/framework/http/router"
+)
+
+func GetRoutes() *[]router.Route {
+	return &routes
+}
+
+var routes = []router.Route{
+	{
+		Path: "/",
+		Handler: func(w http.ResponseWriter, r *http.Request) {
+			fmt.Fprintf(w, "This is the HomePage")
+		},
+	},
+	{
+		Path:    "/ping",
+		Handler: dummy_handler.Ping,
+	},
+	{
+		Path:    "/pong",
+		Handler: dummy_handler.Pong,
+	},
+	{
+		Path:    "/json",
+		Handler: json_handler.JsonResponseHandler,
+	},
+	{
+		Path:    "/test-logger",
+		Handler: dummy_handler.TestLogger,
+	},
```

Comment:
Can you also provide examples for other HTTP methods e.g. POST 

We also need to know how to deal with query parameters

Reply:
structure a bit changed here:
you can take at `internal_module/internal_module.go`. There are some examples there. 

But anyway I would provide a doc for routers and other services when it's design is finished and is not subject to major changes

Reply:
query parameters are still missing in the implemented version of router. It would be the next task